src/site/content/en/blog/optimize-lcp/index.md

```diff
@@ -38,29 +38,90 @@ To provide a good user experience, **sites should strive to have an LCP of 2.5 s
+
+LCP data based on real users can be surfaced from Real User Monitoring (RUM) tools installed on a site, or via the [Chrome User Experience Report (CrUX)](https://developer.chrome.com/docs/crux/) which collect anonymous data from real Chrome users for millions of websites.
+
+### Using PageSpeed Insights CrUX LCP data
+
+[PageSpeed Insights](https://pagespeed.web.dev/) provides access to CrUX data in the top section ("Discover what your real users are experiencing"), and then more detailed lab-based data in the bottom section ("Diagnose performance issues"). Where you have CrUX data, always concentrate on this first.
+
+{% Img src="image/W3z1f5ZkBJSgL1V1IfloTIctbIF3/enRFus2GE24gvchY9fdV.png", alt="CrUX data shown in PageSpeed Insights", width="800", height="478" %}
 
 {% Aside %}
-  For a visual overview of the context presented in this guide, see [A Deep Dive into Optimizing LCP](https://youtu.be/fWoI9DXmpdk) from Google I/O '22:
-  {% YouTube "fWoI9DXmpdk" %}
+Where CrUX does not provide data (for example, a page with insufficient traffic to get page-level data), CrUX should be supplemented with RUM data collected using JavaScript APIs running on the web page. This can also provide much more data than CrUX can expose as a public dataset. Later in this guide we will explain how to collect this data using JavaScript.
 {% endAside %}
 
+PageSpeed Insights shows up to four different CrUX data:
+- **Mobile** data for **This URL**
+- **Desktop** data for **This URL**
+- **Mobile** data for the whole **Origin**
+- **Desktop** data for the whole **Origin**
+
+These can be toggled in the controls at the top, and top right-hand side of this section. Be aware that where a URL does not have sufficient data to be shown at URL-level, but does have data for the origin, PageSpeed Insights will automatically show this.
+
+{% Img src="image/W3z1f5ZkBJSgL1V1IfloTIctbIF3/wWQxf1DbjElItJTcJsBn.png", alt="PageSpeed Insight's falling back to origin-level data where url-level data is not available", width="800", height="295" %}
+
+The LCP for the whole origin may be very different to an individual page's LCP depending on how the LCP is loaded on that page compared to other pages on that origin. It can also be affected by how visitors navigate to these pages. Home pages tend to be visted by new users and so may often be loaded "cold", without any cached content and so are often the slowest pages on a website.
+
+Looking at the four difference pieces of CrUX data can help you understand whether an LCP issue is specific to this page, or a more general site-wide issue. Similarly, it can show which device types have LCP issues.
```

Comment:
```suggestion
Looking at the four different categories of CrUX data can help you understand whether an LCP issue is specific to this page, or a more general site-wide issue. Similarly, it can show which device types have LCP issues.
```